content/news/ragls5-and-team-tournament.md

```diff
@@ -0,0 +1,20 @@
+<div style="text-align:center" markdown="1">
+![Red Alert Global League Season 5](/images/news/20180317-ragl-V.png)
+</div>
+
+Greetings, commanders!
+
+Red Alert Global League is rolling yet again, with Season 5 starting on April 2nd. Masters division is as strong as ever this time, with several players returning to top tier and new promising contestants rising up as well; Minions are split into 2 groups as the single division wasn't able to hold that many good players. To raise the stakes even higher the map pool now holds mostly new competitive maps so that players are challenged to improvise and develop new tactics and strategies on the battlefields.
```

Comment:
Not sure about "top tier" alone, feels like it is missing a "play" (returing to top tier play). Might just be me though.

Maybe replace the second "hold" with "contains" to avoid using that twice directly in succession. I'm no linguist though, so not really a blocker. ^^